bikeshed/cli.py

```diff
@@ -620,7 +621,7 @@ def handleRefs(options, extras):
         exact=options.exact,
     )
     if constants.printMode == "json":
-        p(json.dumps(refs, indent=2, default=config.getjson))
+        p(json.dumps(refs, indent=2, default=getjson))
```

Comment:
`getjson` is a method off printjson, so adjusted with new import above. Not sure if this needs a test of some sort

Reply:
This should be fine. It's clear that I don't use this codepath. ^_^